webpack.config.js

```diff
@@ -6,52 +6,37 @@ Encore
     .cleanupOutputBeforeBuild()
     // Convert typescript files.
     .enableTypeScriptLoader()
-    .autoProvidejQuery()
```

Comment:
No issues with removing this? :+1:

Reply:
I guess the diff is a bit weird but, autoProvideJQuery() is still there. Its moved a bit down in the file.